Error: Cannot find module 'express'

Unsure if I am missing something, but this project does not run at present

```
$ dc up
Starting node-example_node_1 ... done
Attaching to node-example_node_1
node_1  | yarn run v1.22.5
node_1  | $ node index.js
node_1  | internal/modules/cjs/loader.js:883
node_1  |   throw err;
node_1  |   ^
node_1  |
node_1  | Error: Cannot find module 'express'
node_1  | Require stack:
node_1  | - /app/index.js
node_1  |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
node_1  |     at Function.Module._load (internal/modules/cjs/loader.js:725:27)
node_1  |     at Module.require (internal/modules/cjs/loader.js:952:19)
node_1  |     at require (internal/modules/cjs/helpers.js:88:18)
node_1  |     at Object.<anonymous> (/app/index.js:1:17)
node_1  |     at Module._compile (internal/modules/cjs/loader.js:1063:30)
node_1  |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
node_1  |     at Module.load (internal/modules/cjs/loader.js:928:32)
node_1  |     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
node_1  |     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
node_1  |   code: 'MODULE_NOT_FOUND',
node_1  |   requireStack: [ '/app/index.js' ]
node_1  | }
node_1  | error Command failed with exit code 1.
node_1  | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
node-example_node_1 exited with code 1
```

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
-    "express": "^4.15.3"
+    "express": "^4.17.1"
   },
   "scripts": {
     "start": "node index.js"

